charts/chronicle/templates/test-token-getter-roles.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.test.enabled }}
+{{- if .Values.auth.required }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "common.names.fullname" . }}-test-token-getter-role
+  namespace: {{.Release.Namespace}}
+rules:
+  - apiGroups:
+      - "" # "" refers to the core API group
+    resources:
+      - pods
+      - pods/log
+      - pods/exec
+    verbs:
+      - create
```

Comment:
you don't want to be able to create pods

Reply:
```
Getting token from id-provider ...
Error from server (Forbidden): pods "test-chronicle-test-id-provider-0" is forbidden: User "system:serviceaccount:default:test-chronicle" cannot create resource "pods/exec" in API group "" in the namespace "default"
```